### ISSUE TITLE
Java org.brotli.dec sources.jar is not properly structured.

### ISSUE THREAD

**Issue by lambdaupb** [google/brotli#516]
The sources jar file contains the source files, but all at the top-level instead of the package path.

currently:
```
> unzip -l dec-0.1.0-sources.jar
Archive:  dec-0.1.0-sources.jar
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  02-20-2017 15:08   META-INF/
      132  02-20-2017 15:08   META-INF/MANIFEST.MF
     5066  02-20-2017 14:06   BitReader.java
     5208  02-20-2017 14:06   BrotliInputStream.java
      526  02-16-2017 16:46   BrotliRuntimeException.java
     7835  02-16-2017 16:46   Context.java
    31028  02-20-2017 14:06   Decode.java
   243079  02-20-2017 14:06   Dictionary.java
     4256  02-16-2017 16:46   Huffman.java
     1416  02-16-2017 16:46   HuffmanTreeGroup.java
     1521  02-16-2017 16:46   Prefix.java
      491  02-16-2017 16:46   RunningState.java
     3385  02-20-2017 14:05   State.java
     8901  02-20-2017 14:05   Transform.java
     1535  02-16-2017 16:46   Utils.java
     1155  02-16-2017 16:46   WordTransformType.java
---------                     -------
   315534                     16 files
```

But it should be structured by package like the compiled jar:

```
> unzip -l dec-0.1.0.jar
Archive:  dec-0.1.0.jar
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  02-20-2017 15:08   META-INF/
      132  02-20-2017 15:08   META-INF/MANIFEST.MF
        0  02-20-2017 15:08   org/
        0  02-20-2017 15:08   org/brotli/
        0  02-20-2017 15:08   org/brotli/dec/
     3479  02-20-2017 15:08   org/brotli/dec/BitReader.class
     3003  02-20-2017 15:08   org/brotli/dec/BrotliInputStream.class
      682  02-20-2017 15:08   org/brotli/dec/BrotliRuntimeException.class
    11908  02-20-2017 15:08   org/brotli/dec/Context.class
     1043  02-20-2017 15:08   org/brotli/dec/Decode$1.class
    16233  02-20-2017 15:08   org/brotli/dec/Decode.class
     1323  02-20-2017 15:08   org/brotli/dec/Dictionary$DataHolder.class
    49485  02-20-2017 15:08   org/brotli/dec/Dictionary$DataHolder0.class
    46155  02-20-2017 15:08   org/brotli/dec/Dictionary$DataHolder1.class
    51863  02-20-2017 15:08   org/brotli/dec/Dictionary$DataHolder2.class
     1313  02-20-2017 15:08   org/brotli/dec/Dictionary.class
     2007  02-20-2017 15:08   org/brotli/dec/Huffman.class
      982  02-20-2017 15:08   org/brotli/dec/HuffmanTreeGroup.class
     1631  02-20-2017 15:08   org/brotli/dec/Prefix.class
     1653  02-20-2017 15:08   org/brotli/dec/RunningState.class
     3376  02-20-2017 15:08   org/brotli/dec/State.class
     4980  02-20-2017 15:08   org/brotli/dec/Transform.class
      952  02-20-2017 15:08   org/brotli/dec/Utils.class
     2306  02-20-2017 15:08   org/brotli/dec/WordTransformType.class
        0  02-20-2017 15:08   META-INF/maven/
        0  02-20-2017 15:08   META-INF/maven/org.brotli/
        0  02-20-2017 15:08   META-INF/maven/org.brotli/dec/
     2470  02-20-2017 14:42   META-INF/maven/org.brotli/dec/pom.xml
       98  02-20-2017 15:08   META-INF/maven/org.brotli/dec/pom.properties
---------                     -------
   207074                     29 files
```

This makes the IDE not find the source files which would be very helpful for seeing javadoc and debugging.

**Comment by eustas** [comment]
Thanks for the report. Going to fix it soon.

**Comment by eustas** [comment]
Released 0.1.1 with fixed structure of sources.jar
Should become available in less than 10 hours.
